src/main/blackbox/blackbox.c

```diff
@@ -1429,10 +1429,11 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE("dshot_bidir", "%d",                     motorConfig()->dev.useDshotTelemetry);
 #endif
 #ifdef USE_RPM_FILTER
-        BLACKBOX_PRINT_HEADER_LINE("gyro_rpm_notch_harmonics", "%d",        rpmFilterConfig()->gyro_rpm_notch_harmonics);
-        BLACKBOX_PRINT_HEADER_LINE("gyro_rpm_notch_q", "%d",                rpmFilterConfig()->gyro_rpm_notch_q);
-        BLACKBOX_PRINT_HEADER_LINE("gyro_rpm_notch_min", "%d",              rpmFilterConfig()->gyro_rpm_notch_min);
-        BLACKBOX_PRINT_HEADER_LINE("rpm_notch_lpf", "%d",                   rpmFilterConfig()->rpm_lpf);
+        BLACKBOX_PRINT_HEADER_LINE("rpm_filter_harmonics", "%d",            rpmFilterConfig()->rpm_filter_harmonics);
```

Comment:
See below. Changing this will just create work in blackbox log viewer maintenance, and add code bloat there.

Reply:
I'm doing that maintenance now, so it is not difficult to make this change at this point in time.